packages/client/lib/rpc/modules/engine.ts

```diff
@@ -748,8 +759,42 @@ export class Engine {
   }
 
   private async forkchoiceUpdatedV2(
-    params: [forkchoiceState: ForkchoiceStateV1, payloadAttributes: PayloadAttributesV2 | undefined]
+    params: [
+      forkchoiceState: ForkchoiceStateV1,
+      payloadAttributes: PayloadAttributesV1 | PayloadAttributesV2 | undefined
+    ]
   ): Promise<ForkchoiceResponseV1 & { headBlock?: Block }> {
+    console.log(params)
```

Comment:
console.log

Reply:
Ah right the linter warns instead of throws